src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -10,6 +10,10 @@
     <Company>Microsoft Corporation</Company>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <DefineConstants>NETSTANDARD20</DefineConstants>

```

Comment:
No need to define this yourself. When targeting netstandard2.0, the compiler automatically defines a NETSTANDARD2_0 constant.